addons/sale/static/src/js/sale_action_helper/sale_action_helper_dialog.xml

```diff
@@ -2,27 +2,30 @@
 
 <templates xml:space="preserve">
     <t t-name="sale.SaleActionHelperDialog">
-        <Dialog
-            bodyClass="'shadow'"
-            contentClass="'border-0 bg-transparent shadow-none'"
-            footer="false"
-            size="'xl'"
-            technical="false"
-            withBodyPadding="false"
-        >
-            <div class="ratio ratio-16x9">
-                <iframe
-                    allow="autoplay; encrypted-media; picture-in-picture; web-share"
-                    width="1140"
-                    height="641"
-                    t-att-src="this.props.url"
-                    title="Sale"
-                    frameborder="0"
-                    allowfullscreen="1"
-                >
-                    Your browser does not support iframe.
-                </iframe>
-            </div>
-        </Dialog>
+        <div t-on-click.stop="close">
+            <Dialog
+                bodyClass="'shadow'"
+                contentClass="'border-0 bg-transparent shadow-none'"
+                footer="false"
+                size="'xl'"
+                technical="false"
+                withBodyPadding="false"
+            >
+                <div class="ratio ratio-16x9">
+                    <iframe
+                        allow="autoplay; encrypted-media; picture-in-picture; web-share"
+                        width="1140"
+                        height="641"
+                        t-att-src="this.props.url"
+                        title="Sale"
+                        frameborder="0"
+                        allowfullscreen="1"
+                        t-on-click.stop=""
+                    >
+                        Your browser does not support iframe.
+                    </iframe>
+                </div>
+            </Dialog>
+        </div>
```

Comment:
why you need to add div? :thinking: 
you can directly write `t-on-click.stop="props.close"` on the Dialog tag

Reply:
Sorry, accidentally I pushed the different patch. Updated the changes :+1: